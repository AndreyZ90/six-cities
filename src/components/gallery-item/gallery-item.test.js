import React from 'react';
import TestRenderer from 'react-test-renderer';

import GalleryItem from '@/components/gallery-item/gallery-item';

const src = `img/hotel.jpg`;

describe(`GalleryItem component snapshot`, () => {
  test(`Should correctly render GalleryItem component`, () => {
    const tree = TestRenderer.create(<GalleryItem src={src} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
