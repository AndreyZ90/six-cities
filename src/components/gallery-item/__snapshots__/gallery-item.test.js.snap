// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GalleryItem component snapshot Should correctly render GalleryItem component 1`] = `
<div
  className="property__image-wrapper"
>
  <img
    alt="Photo studio"
    className="property__image"
    src="img/hotel.jpg"
  />
</div>
`;
